infoenergia_api/contrib/contracts.py

```diff
@@ -93,67 +98,55 @@ def tariffHistory(self):
             }
             for modcon in find_changes(self._erp, self.modcontractuals_ids, 'tarifa')]
 
+
     @property
-    def currentPower(self):
+    def power(self):
         """
-        Current power:
-        "power_": {
-          "power": 123,
-          "dateStart": "2014-10-11T00:00:00Z",
-          "dateEnd": null,
-          "measurement_point": '05'
+        power:
+         {
+            "P1": 20000,
+            "P2": 20000,
+            "P3": 20000
         }
         """
-        modcon = find_changes(self._erp, self.modcontractual_activa[0], 'potencia')[-1]
+        if '2.' in self.tarifa:
+            return {}
+        period_obj = self._erp.model('giscedata.polissa.potencia.contractada.periode')
+        period_powers = period_obj.read([('polissa_id', '=', self.id)])
+
         return {
-            "power": int(modcon['potencia'] * 1000),
-            "dateStart": make_utc_timestamp(modcon['data_inici']),
-            "dateEnd": make_utc_timestamp(modcon['data_final']),
-            "measurement_point": modcon['agree_tipus']
+            'P{}'.format(
```

Comment:
Hello! Thank you for your review!

There is an other pending development related with prices that will need a refactor so I think it will be better to take advantage of that other development to move that code to an external function (and do some other changes) :)